v1/pipeline/steps/custom/template-link-config.yaml

```diff
@@ -1,17 +1,12 @@
 title: TemplateLinkConfig
 type: object
 required:
-- templateRef
+- ref
 properties:
-  templateInputs:
+  inputs:
     $ref: ../../common/json-node.yaml
-  templateRef:
+  ref:
     type: string
-  templateVariables:
+  variables:
```

Comment:
why template link has variables? and isnt variables def changed and not json node anymore

Reply:
removed variables, inputs will solve the use case